src/generate/renders/TemplateBase.ts

```diff
@@ -33,3 +33,24 @@ export abstract class TemplateBase {
 
     public abstract async GetRenderData(model: CodeModelAz): Promise<any>;
 }
+
+export class SimpleTemplate extends TemplateBase {
```

Comment:
Just curious why don't we put it as a single file ?

Reply:
I'm neutral on whether need to put it in a single file.
I put it in this file since I think this is an common render which may be used for many .njx